spec/queries/video_hearing_day_request_type_query_spec.rb

```diff
@@ -79,6 +79,42 @@
       end
     end
 
+    context "single ama virtual hearing that has been held" do
+      let(:hearing_day) do
+        create(
+          :hearing_day,
+          regional_office: "RO01",
+          request_type: HearingDay::REQUEST_TYPES[:video],
+          scheduled_for: Time.zone.today - 1.week
+        )
+      end
+      let(:hearing) do
+        create(
+          :hearing,
+          :held,
+          hearing_day: hearing_day
+        )
+      end
+      # Virtual hearing has already been cleaned up successfully.
+      let!(:virtual_hearing) do
+        create(
+          :virtual_hearing,
+          :initialized,
+          :all_emails_sent,
+          conference_deleted: true,
+          hearing: hearing,
+          status: :active
+        )
+      end
+
+      it "sanity check: hearing is virtual" do
+        expect(hearing.reload.virtual?).to be true
+      end
+
+      include_examples "it returns correct hash with expected key and value",
+                       "Virtual"
+    end
```

Comment:
```suggestion

      context "with a legacy hearing" do
        let(:disposition) { Constants.HEARING_DISPOSITION_TYPES.held }
        let(:hearing) { create(:legacy_hearing, disposition: disposition, hearing_day: hearing_day) }

        include_examples "it returns correct hash with expected key and value",
                         "Virtual"
      end
    end
```

I added this test for the `LegacyHearing` case when I was reviewing.